src/main/java/com/xebialabs/restorant/conditions/ThreeTierCondition.java

```diff
@@ -0,0 +1,17 @@
+package com.xebialabs.restorant.conditions;
+
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+/**
+ * Created by atiwary on 23/01/17.
+ */
+public class ThreeTierCondition implements Condition{
```

Comment:
could we name this class to suggest the functional aspect of this implementation